apps/meteor/client/hooks/useAppActionButtons.ts

```diff
@@ -42,6 +43,24 @@ export const useAppActionButtons = <TContext extends `${UIActionButtonContext}`>
 		[],
 	);
 
+	useEffect(() => {
```

Comment:
You can use `useStatusConnection` hook for this. It's pretty much the same thing